tests/test_kernels.py

```diff
@@ -84,13 +98,17 @@ def test_conservation(grid_type_field_and_extra_kwargs):
     #   --> test with area = 1
     # - all other Laplacians  act on potentially irregular grid
     #   --> need area information
-    if not issubclass(LaplacianClass, AreaWeightedMixin):
-        for k, v in extra_kwargs.items():
-            if "area" in k:
-                area = v
-                break
-    res = laplacian(data) * area
-    np.testing.assert_allclose(res.sum(), 0.0, atol=1e-12)
+    if issubclass(LaplacianClass, AreaWeightedMixin):
+        area = 1
+    else:
+        area = extra_kwargs.get("area", None)
+        if area is None:
+            area = extra_kwargs.get("tarea", 1)
+
+    res = laplacian(data)
+
+    # currently failing only for TRIPOLAR_POP_WITH_LAND. Why?
+    np.testing.assert_allclose((area * res).sum(), 0.0, atol=1e-12)
```

Comment:
Any idea why this test would fail only for TRIPOLAR_POP_WITH_LAND?

Reply:
See https://github.com/ocean-eddy-cpt/gcm-filters/pull/80#issue-701806629